### PR TITLE
ui: improve sidebar panels

### PR DESCRIPTION
**Pull request by eonpatapon**
Multiple improvements to the side bar:

 * introduced a panel component to easily add
   sidebar blocks
 * introduced a generic collapse component
 * removed specific code from object-detail component
 * removed unused CSS classes

Also upgrade to Vuejs 2.5.16

### CODE REVIEW

**Comment by safchain** [comment]
@eonpatapon I really like what you did with this PR, do you think that you will have time to work on it soon ?

**Comment by eonpatapon** [comment]
@safchain I was having issues with selenium tests. I think that at some point in the capture test selenium wants to click on a tab but there is a notification that is hidding this tab at the same time

**Comment by safchain** [comment]
@eonpatapon ok I will have a look for another test with selenium we click on the notification to close it before clicking the tab, I'll try to adapt to this test.

**Comment by safchain** [comment]
rerun tests

File: vendor/vendor.json [hunk 1]
```diff
@@ -1101,7 +1101,7 @@
 			"revisionTime": "2017-11-28T22:01:57Z"
 		},
 		{
-			"checksumSHA1": "ORc1X58pM+oNj08y5wXask2sV4c=",
+			"checksumSHA1": "Uvt0pTvpmwUw0EwDiJm0AiTd754=",
```
Comment (safchain) [inline vendor/vendor.json]: doesn't seem to be related to this patch

File: vendor/vendor.json [hunk 2]
```diff
@@ -1077,7 +1077,7 @@
 			"versionExact": "v1.0.0"
 		},
 		{
-			"checksumSHA1": "GENxfNGiSzB9hzo2fPZkI4F/Zzg=",
+			"checksumSHA1": "me40nSy9NBr9MmAwe5vDvTgSVZs=",
```
Comment (safchain) [inline vendor/vendor.json]: does this patch need to update Go dependencies ? if not mandatory I would prefer to not update

**Comment by eonpatapon** [comment]
@safchain removed changes done to vendor.json. I don't know why but it happens all the time without changing deps

**Comment by safchain** [comment]
@eonpatapon maybe you govendor binary is not up-to-date

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by safchain** [comment]
@eonpatapon thx ! that's really nicer
